config/prod/recover-synapse-attached-input-bucket.yaml

```diff
@@ -0,0 +1,42 @@
+# Provision a general S3 bucket or a Synapse External Bucket
+# http://docs.synapse.org/articles/custom_storage_location.html
+
+# For use in RECOVER Synapse project (syn51105296)
+
+#### 
+# NOTE:
+# This is the second bucket in the ingress pipeline for RECOVER project. We will create a copy of the first bucket (recover_input_shared_bucket).
+# This bucket is to make sure we have a data source that is constant and cannot be edited by externals 
+# Only Sage has access to this bucket, and this will be linked to the RECOVER Synapse project (syn51105296).
+# The file list will be replicated in Synapse via STS token 
+#(https://help.synapse.org/docs/Compute-Directly-on-Data-in-Synapse-or-S3.2048426057.html#ComputeDirectlyonDatainSynapseorS3-Synapse-ManagedSTSStorageLocations)
+####
+template:
+  type: "http"
+  url: "https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.2.7/s3-bucket.yaml"
+stack_name: "recover-synapse-attached-input-bucket"
+stack_tags:
+  Department: "SysBio"
+  Project: "RECOVER"
+  OwnerEmail: 'synapse-service-sysbio-recover-data-storage-01@sagebase.org'
+  # This is the shared account (credentials shared on Last Pass) for RECOVER
+  CostCenter: "MGH RECOVER / 122500"
+parameters:
+  # The Sage department (Platform, CompOnc, SysBio, Governance, etc.,)
+  Department: "SysBio"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc.,)
+  Project: "RECOVER"
+  # The resouce owner
+  OwnerEmail: 'synapse-service-sysbio-recover-data-storage-01@sagebase.org'
+  # This is the shared account (credentials shared on Last Pass) for RECOVER
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: 'true'
+  # (Optional) Synapse username (default: ""), required if AllowWriteBucket=true
+  SynapseUserName: 'meghasyam'
```

Comment:
After thinking a little about this.  Were you thinking of actually creating entities in Synapse or doing something similar to what we do for the parquet datasets in BridgeDowstream (access via STS)?

The reason I ask is because `SynapseUserName` is specified for if you want to create file entities in Synapse that link to S3 objects in this bucket. (I have the question for the other bucket as well). Happy to hop on a quick slack huddle Monday.